src/Microsoft.VisualStudio.Editors/PropPages/DebugPropPage.resx

```diff
@@ -984,6 +984,12 @@
   <metadata name="$this.Localizable" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
     <value>True</value>
   </metadata>
+  <data name="$this.AutoScaleDimensions" type="System.Drawing.SizeF, System.Drawing">
+    <value>6, 13</value>
```

Comment:
Why is the default height and width set at 6 and 13? I see varying value of height through out the code base and pretty much 13 for the width though.